packages/atomic/src/components/atomic-context-provider/atomic-context-provider.tsx

```diff
@@ -1,19 +1,22 @@
 import {Component, Prop} from '@stencil/core';
-import {buildContext, Engine} from '@coveo/headless';
-import {Initialization} from '../../utils/initialization-utils';
+import {buildContext} from '@coveo/headless';
```

Comment:
Not really related to this PR, but this atomic component should be eliminated.

Instead (like we discussed in a previous planif), I would propose to remove all "non visual" component in atomic.

To be able to achieve things like passing context, we would instead make headless engine easily reachable from atomic for custom code.

So that users could do stuff like : 

```
const engine =  document.querySelector('#search').engine;
const myContext = document.querySelector('#search').headless.buildContext(engine);
myContext.set({...})
```

Reply:
https://coveord.atlassian.net/browse/KIT-348